Active Member Requirements (#2)
BUUPE/Governing-Documents: Bylaws.md

***Date of Charter Ceremony: May 2018***

# Bylaw I: Scope of the Bylaws

1. These bylaws shall govern the conduct of the chapter in those matters not specifically treated in the Constitution.
2. Provisions of these bylaws shall be subordinate to those of the Constitution.

# Bylaw II: Election of Officers

1. Election of officers shall be held during a chapter meeting in the middle of each Spring semester. 
2. Newly elected officers shall be brought up to date for the remainder of the Spring semester and will begin to serve on the last meeting of the Spring semester. In order to be eligible to hold office, the President must have been active in the chapter for at least one semester.
3. The officers of this chapter shall be elected by secret ballot and there shall be no voting of proxies allowed.

# Bylaw III: Meetings

1. Chapter meetings will be held as scheduled during the Spring and Fall semesters.
2. Special meetings may be called by the President, the Executive Council, or by petition of one-half of the active membership of the chapter.

> 3. The order of conduct of a Chapter meeting shall be as follows:

Author: ```suggestion
3. The order of conduct of a chapter meeting shall be as follows:
```

A. Call to order

> B. Attendance in

Author: ```suggestion
B. Attendance
```

6. Initiation of new members may be conducted at a regular meeting or at a special meeting called for the purpose.
7. Notification of all active members shall be given by the Secretary one week prior to a special meeting. The notification shall state the purpose of the meeting.
 
# Bylaw IV: Active Membership Requirements

1. An Active member is required to earn 20 points every semester, and 6 of these points need to be from recruitment activities. These points can earned in the following ways:

| Activity  | Recruitment | Point Value  |
| ------------- |:-------------:| -----:|
| Chapter      | False | 1 |
| Social Event  | False      |  1 |
| Professional Event | False   | 2 |
| UPE Project | False   | Varies |
| Speaking in front of class | True   | 1 |
| Recruitment Event | True   | 2 |

1. If a member does not meet the active member requirements, they will be considered inactive. 

> 2. If a member is considered inactive for two continuous semesters, they will be removed from Massachusetts Eta Chapter of Upsilon Pi Epsilon.

Author: ```suggestion
2. If a member is considered inactive for two continuous semesters, they will be removed from the Massachusetts Eta Chapter of Upsilon Pi Epsilon.
```

# Bylaw VI: Initiation

1. The initiation of new members shall be held at the end of their onboarding period, which shall be no later than two weeks before the end of the semester.
2. The initiation ceremony shall specifically avoid any practice which conflicts with Boston University regulations or which constitutes an attack on the personal dignity of the initiates. The activities shall be devised to stimulate the candidate’s interest in computer science and to acquaint them better with members of the chapter.

# Bylaw VII: Election of Members

1. There shall be two elections of new members each year.
2. The Recruitment Team shall organize marketing and information events in order to find potential applicants.
3. All interested applicants will attend the required events as stated in the chapter constitution.
4. The interested applicants shall apply for membership in accordance to the application process as stated in the chapter constitution.
5. The list of eligible candidates, together application and interview remarks shall be submitted to all eligible voting members of the chapter.

> 6. A vote of simple majority is needed for an eligible candidate to be elected for full membership.

Author: ```suggestion
6. An affirmative vote of 3/4 or greater is needed for an eligible candidate to be elected for full membership.
```

7. The names of those elected to membership shall be given to the Secretary, who shall then issue invitations to become candidate members of the Upsilon Pi Epsilon National Association, as well as the Massachusetts Eta Chapter.
8. Failure of a candidate member-elect to notify the chapter of his or her acceptance of the invitation to membership within two weeks of its issuance shall constitute notice of refusal.

# Bylaw VIII: Adoption and Amendment

1. These bylaws shall become effective immediately upon adoption.

> 2. These bylaws shall remain in effect until revised or amended by a two-thirds affirmative vote of the active membership of this Chapter

Author: ```suggestion
2. These bylaws shall remain in effect until revised or amended by a two-thirds affirmative vote of the active membership of this Chapter.
```
